Shared/Managers/Manga/MangaManager.swift

```diff
@@ -253,12 +253,23 @@ extension MangaManager {
 
                             // update chapter list
                             if mangaObject.chapters?.count != chapters.count && !chapters.isEmpty {
-                                CoreDataManager.shared.setChapters(
+                                let newChapters = CoreDataManager.shared.setChapters(
                                     chapters,
                                     sourceId: manga.sourceId,
                                     mangaId: manga.id,
                                     context: context
                                 )
+                                // update manga updates
+                                for chapter in newChapters
+                                where mangaObject.langFilter != nil ? chapter.lang == mangaObject.langFilter : true
```

Comment:
not sure if we should do this

Reply:
I think we should, this is good. people that have a chapter language filter set probably don't change it often, and probably only care about the chapters for that particular language anyways